oidc-ui/Dockerfile

```diff
@@ -5,12 +5,20 @@ ENV SIGN_IN_WITH_ESIGNET_PLUGIN_URL=$sign_in_with_esignet_plugin_url
 
 # Set a build-time environment variable (replace YOUR_ENV_VARIABLE_NAME with the desired variable name)
 ARG oidcUIPublicUrl
+ARG uiDefaultTheme
```

Comment:
Can we think of having the entire public directory as a mountable customizable theme?

Reply:
All changes are now restricted to the public folder.